dspace-uclouvain/src/main/java/org/dspace/uclouvain/degreeMappers/DegreeMappersService.java

```diff
@@ -34,9 +35,9 @@ public class DegreeMappersService {
      */
     @SuppressWarnings("unchecked")
     DegreeMappersService() {
-        this.fileLoader = (ConfigurationFile<DegreeMapper>) this.configurationFileFactory.getConfigurationFile(DegreeMappersConfigurationFile.NAME);
```

Comment:
You can the remove the `public static final String NAME` in the `DegreeMappersConfigurationFile` class since it is not used anymore.